init.lua

```diff
@@ -118,8 +118,26 @@ local function add_ore(modname, description, mineral_name, oredef)
 			description = S("%s Block"):format(S(description)),
 			tiles = { img_base .. "_block.png" },
 			groups = {snappy = 1, bendy = 2, cracky = 1, melty = 2, level= 2},
-			sounds = default_metal_sounds,
+			sounds = default.node_sound_metal_defaults(),
 		})
+		if minetest.get_modpath("moreblocks") then	
+			stairsplus:register_all("moreores", mineral_name.."_block", block_item, {
+				description = S("%s Block"):format(S(description)),
+				tiles = { img_base .. "_block.png" },
+				groups = {snappy = 1, bendy = 2, cracky = 1, melty = 2, level= 2},
+				sounds = default.node_sound_metal_defaults()
+			})
+		elseif minetest.get_modpath("stairs") then
+			stairs.register_stair_and_slab(
```

Comment:
When minetest/minetest_game/pull/2584 will be merged, `desc_stair_outer` and `desc_stair_inner` parameters could be added (with translations for `@1 Outer Stair` and `@1 Inner Stair` in `locale` directory)